### PR TITLE
Initialized reset password form

### PR DESCRIPTION
**Pull request by aidenschrock**

### CODE REVIEW

**Review by Isaiah-Hamilton — COMMENTED** [review]
we are using `yarn.lock` instead of `package-lock.json`

**Comment by aidenschrock** [comment]
Fixed, now using yarn

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+}
```
Comment (VictorPeralta) [inline .vscode/settings.json]: Code editor settings should be ignored

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "private": true,
   "license": "gpl-3.0",
   "scripts": {
-    "dev": "next dev",
+    "dev": "next",
```
Comment (VictorPeralta) [inline package.json]: Why did this command change?

**Review by VictorPeralta — DISMISSED** [review]

**Review by Isaiah-Hamilton — APPROVED** [review]

**Pull request merged**